README.md

```diff
@@ -99,6 +99,11 @@ export ASSUMED_ROLE="prod"
 # eval $(assume-role prod)
 ```
 
+Create an alias to avoid typing ```eval $(assume-role prod)```     
+```alias assume-role='function(){eval $(assume-role $@);}'```
```

Comment:
If done this way, the alias will be actually calling the function every time you start a session or open a new terminal. If you have MFA enabled this could be a bit annoying and not desired if you would like to manage multiple roles.

Reply:
I don't think so ```alias assume-role='function(){eval $(assume-role $@);}'``` is not executed it's just a wrapper for ```eval $(assume-role prod)```

Reply:
```eval $(assume-role $@)```is not executed until you call the alias

Reply:
basically its an alias to an anonymous function

Reply:
You are right, I tested it before and it actually triggered. But now it's working for me, so it's a good comment to be included there.